.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
```

Comment:
[Required] :recycle:      Please ensure that you make this Readme descriptive. This is the first landing of users as well as other people who may want to see your project. Also give acknowledgement if anyone's code was used. This ensures that you avoid potential copyright problems.